.github/workflows/instrumentations_1.yml

```diff
@@ -38,10 +38,13 @@ jobs:
           - "resource-detector-azure"
           - "resource-detector-container"
           - "util-http"
+          - "fastapi-slim"
         os: [ubuntu-20.04]
         exclude:
           - python-version: pypy3
             package: "prometheus-remote-write"
+          - python-version: pypy3
```

Comment:
It doesn't work with pypy?

Reply:
Could you please try if it works with the changes @xrmx suggested?

Reply:
yeah, I did and it works.